query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/order_and_pagination/order_by_dependent.rs

```diff
@@ -92,8 +92,14 @@ mod order_by_dependent {
                 }
               }
             }"#,
-            Postgres(_) => r#"{"data":{"findManyModelA":[{"id":1,"b":{"id":1}},{"id":2,"b":{"id":2}},{"id":3,"b":null}]}}"#,
-            _ => r#"{"data":{"findManyModelA":[{"id":3,"b":null},{"id":1,"b":{"id":1}},{"id":2,"b":{"id":2}}]}}"#
+            CockroachDb => vec![
+                // NULLS FIRST
+                r#"{"data":{"findManyModelA":[{"id":3,"b":null},{"id":1,"b":{"id":1}},{"id":2,"b":{"id":2}}]}}"#,
+                // NULLS LAST
+                r#"{"data":{"findManyModelA":[{"id":1,"b":{"id":1}},{"id":2,"b":{"id":2}},{"id":3,"b":null}]}}"#
+            ],
```

Comment:
I'm not sure I understood why we need to match against both `NULLS FIRST` and `NULLS LAST`.

I assume we run our tests against a single version of cockroach now, right?
- If yes, then we should only expect either `NULLS FIRST` or `NULLS LAST` depending on Cockroach's default
- If no, then that's ok, but I don't understand when those defaults have changed

Reply:
I agree, we stick with either one, not both.

Reply:
Similar question asked here: https://prisma-company.slack.com/archives/C01CSQXTNJ3/p1639574313086600?thread_ts=1639511583.079100&cid=C01CSQXTNJ3 I do not understand why we have to cover both cases at the same time - if a future update changes the behavior, we update the tests again.